src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -124,11 +128,15 @@ const MapView: React.FC<MapViewProps> = ({
 
   useEffect(() => {
     if (mapElement && treePopupElement) {
+      const defaultZoom: number = location.state ? location.state.zoom : 12;
+
       LOADER.load()
         .then(() => {
           map = new google.maps.Map(mapElement, {
-            center: BOSTON,
-            zoom: 12,
+            center: location.state
+              ? { lat: location.state.lat, lng: location.state.lng }
```

Comment:
Not too familiar with useLocation. Is it possible to set the initial state to BOSTON so we can avoid this ternary operator?